Assets/__Scripts/MapEditor/Grid/Collections/EventsContainer.cs

```diff
@@ -18,13 +19,33 @@ public bool RingPropagationEditing
         set
         {
             ringPropagationEditing = value;
-            ringPropagationLabels.SetActive(value);
-            eventGridLabels.SetActive(!value);
+            PlatformDescriptor pD = FindObjectOfType<PlatformDescriptor>();
+            labels.UpdateLabels(value, value ? (pD.BigRingManager?.rings.Length ?? 15)+1 : 16);
+            eventPlacement.SetGridSize(value ? (pD.BigRingManager?.rings.Length ?? 15) + 1 : 6 + pD.LightingManagers.Count(s => s != null));
```

Comment:
For environments without rings (K/DA, Glass Desert, etc.), it would be better to resize the grid so that just "All Rings" is visible.

Pretty much what I'm asking is to make the default size be 0 (before adding 1 for "All Rings").